base/gfsppieceop/pieceop.go

```diff
@@ -12,11 +12,11 @@ type GfSpPieceOp struct {
 }
 
 func (p *GfSpPieceOp) SegmentPieceKey(objectID uint64, segmentIdx uint32) string {
-	return fmt.Sprintf("%d_s%d", objectID, segmentIdx)
+	return fmt.Sprintf("seg_%d_s%d", objectID, segmentIdx)
```

Comment:
Shall we use a byte as the prefix to reduce the key size?

Reply:
it does not cost more if we use the long prefix, what's your concern for the key size